pkg/features/_topics.json

```diff
@@ -0,0 +1,20 @@
+{
+  "active": [
+    "config",
```

Comment:
let's also document what a "topic" is and give some guidelines about creating new topics (we can chat offline about this)